Tests/MFBBindingTests.m

```diff
@@ -692,4 +809,603 @@ - (void)test_BindingAssertion_NoBindingsForKeyPathAndBindingAssertionDisabled_Do
 #pragma pop_macro("NS_BLOCK_ASSERTIONS")
 }
 
+
+#pragma mark - Array Binding Test Methods
+
+- (void)test_insertionIntoFirstObjectArray_ChangesSecondObjectArrayWithUpdatedArray
+{
+    NSArray *initialObjects = @[ [NSObject new], [NSObject new], [NSObject new] ];
+
+    _objectA.fineGrainedArrayA = initialObjects;
+
+    _configuration.firstKeyPath = NSStringFromSelector(@selector(fineGrainedArrayA));
+    _configuration.secondKeyPath = NSStringFromSelector(@selector(arrayB));
+    [_configuration setUpBinding];
+
+    NSMutableArray *insertedObjects = [NSMutableArray new];
+    NSMutableIndexSet *indexes = [NSMutableIndexSet new];
+
+    const int ObjectsCount = (int) initialObjects.count;
```

Comment:
Because why not? There's no need for NSNotFound and it still would need to be casted in arc4random_uniform.
Also it's better to avoid unsigned types if you really don't need them.